docs/source/conf.py

```diff
@@ -57,8 +57,7 @@
 html_static_path = ['_static']
 
 # This specifies how many header levels below the title should we include in the table of contents
-# navigation bar. With a navigation depth of 2 we only include the title and the top header of
-# each page.
+# navigation bar.
 html_theme_options = {
-    'navigation_depth': 2,
+    'navigation_depth': 3,
```

Comment:
I think the documentation simply looks better with this naviation_depth simply looks better.